cli_tools

```diff
@@ -10,6 +10,25 @@
 : ${CSCONFIG_FILE:="/azure/ServiceConfiguration.Cloud.cscfg"}
 : ${AZURE_CLI_LOCATION:="/azure"}
 
+_retry() {
+  sleepsec=$1;
+  maxret=$2;
+  shift 2;
+  ret=1;
+  while :; do
+    [[ $ret > $maxret ]] && return 1;
+    echo "${ret}.retry...";
+    eval "$@" && return;
```

Comment:
adding a ` || exitCode=$? ` at the end of eval, and followed by an `echo "---> command failed with exit code: $exitCode" 1>&2 ` would help to debug the issue what causes the retry